TACC/Core-Portal
# FP-1572: Solve Unequal Column Overflow Bug, But Retain 60/40 Layout

## Overview

- Revert https://github.com/TACC/Core-Portal/pull/626 (which changed layout).
- Fix overflow bug by fixing what broke the overflow bug (which retains layout).

## Related

* [FP-1572](https://jira.tacc.utexas.edu/browse/FP-1572)
* expected to be used by https://github.com/TACC/Core-Portal/pull/688

## Changes

- restore removed classes for Dashboard and ManageAccount
- fix mangled global class that was originally used to fix overflow bug

## Testing

1. Verify Dashboard has 60/40 column layout.
1. Verify Dashboard has no [horizontal overflow](https://jira.tacc.utexas.edu/browse/FP-1572).
1. Verify Manage Account has 60/40 column layout.
1. Verify Manage Account has no [horizontal overflow](https://jira.tacc.utexas.edu/browse/FP-1572).
2. (extra) Verify Dashboard and Manage Account at different layouts behave as they had before\*.

<sub>\* Vague on purpose. Sorry. I haven't had time to find the PR that illustrates the full responsive design.</sub>

## UI

| Manage Account (Same Panels) | Dashboard (w/ Custom App) |
| - | - |
| ![Manage Account (No Change)](https://user-images.githubusercontent.com/62723358/186791961-82bd19f6-3ee8-42fe-b4b6-652607a924c4.png) | ![Dashboard With Custom App](https://user-images.githubusercontent.com/62723358/186791964-bcdc77db-823e-4850-b226-7d08571ad8e7.png) |

## Notes

This does not affect other layouts than Dashboard and ManageAccount, because no layouts were using the layouts I've given back to them.

### Improvements that Could be Made

[FP-1603](https://jira.tacc.utexas.edu/browse/FP-1603): Replace unequal columns with CSS multi-column layout with CSS grid. (I should have used CSS Grid when I originally created this layout, but I was not competent enough with it yet.)

https://github.com/TACC/Core-Portal/pull/690#discussion_r956175886: Use something like `count--${panelCount}` to manage small width generically rather than specifically.
